src/ServiceControl.Audit/Infrastructure/Settings/Settings.cs

```diff
@@ -152,6 +153,11 @@ public int MaxBodySizeToStore
 
         public bool EnableFullTextSearchOnBodies { get; set; }
 
+        // Windows services allow a maximum of 125 seconds when stopping a service.
+        // When shutting down or restarting the OS we have no control over the
+        // shutdown timeout
+        public TimeSpan ShutdownTimeout { get; set; } = TimeSpan.FromMinutes(2);
```

Comment:
This only gives 5 seconds of "cancellation" which I think is sufficient because it should be a sort of let everything go and log about it but do not delay termination